### PR TITLE
draft 2 wip

### PR DESCRIPTION
**Pull request by glcohen**

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -87,26 +83,26 @@ figure.table .overlarge {
...
 :: The data template for a credential. Refers to the entirety of a [=Credential Schema=], including both the [=Metadata=] and [=JSON Schema=]. The term may refer to a document either with, or without a [=proof=].
 
 <pre class="example" highlight="json">
 {
-  "type": "https://w3c-ccg.github.io/vc-json-schemas/schema/1.0/schema.json",
-  "modelVersion": "1.0",
-  "id": "did:work:MDP8AsFhHzhwUvGNuYkX7T;id=06e126d1-fa44-4882-a243-1e326fbe21db;version=1.0",
+  "type": "https://w3c-ccg.github.io/vc-json-schemas/schema/2.0/schema.json",
+  "version": "2.0",
+  "id": "did:key:MDP8AsFhHzhwUvGNuYkX7T;id=06e126d1-fa44-4882-a243-1e326fbe21db;version=1.0",
```
Comment (OR13) [inline spec/index.bs:102]: these don't look like valid did keys to me... https://did.key.transmute.industries/

File: spec/index.bs [hunk 2]
```diff
@@ -313,7 +306,7 @@ This section provides the [[JSON_SCHEMA]] definition for [=Credential Schema=] a
   },
   "required": [
     "type",
-    "modelVersion",
+    "version",
```
Comment (OR13) [inline spec/index.bs:309]: I like this change.

File: spec/index.bs [hunk 3]
```diff
@@ -327,10 +320,10 @@ This section provides the [[JSON_SCHEMA]] definition for [=Credential Schema=] a
 <pre class="example" highlight="json">
 {
   "type": "https://w3c-ccg.github.io/vc-json-schemas/schema/1.0/schema.json",
-  "modelVersion": "1.0",
-  "id": "did:work:MDP8AsFhHzhwUvGNuYkX7T;id=06e126d1-fa44-4882-a243-1e326fbe21db;version=1.0",
+  "version": "1.0",
+  "id": "did:key:MDP8AsFhHzhwUvGNuYkX7T;id=06e126d1-fa44-4882-a243-1e326fbe21db;version=1.0",
   "name": "EmailCredentialSchema",
-  "author": "did:work:MDP8AsFhHzhwUvGNuYkX7T",
+  "author": "did:key:MDP8AsFhHzhwUvGNuYkX7T",
```
Comment (OR13) [inline spec/index.bs:326]: might be better to just use `did:example:123`, since its more neutral.

File: spec/index.bs (hunk 1 above)
Comment (glcohen) [reply spec/index.bs:102]: will change to example

**Comment by glcohen** [comment]
@OR13 I have updated a good amount of language and examples in the specification.
I have not yet included a schema for the entire credential. I need to think how to best specify which piece(s) of a credential a schema applies to - I'd like to take that in a follow up PR. Let me know what you think about this cut.

Perhaps it's best to create a v2 branch and push to there instead of master?

**Review by OR13 — APPROVED** [review]

**Pull request merged**